### PR TITLE
Data pipeline cleanup

### PR DESCRIPTION
**Pull request by jjstratton**

### CODE REVIEW

File: DATA.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
+# CIT-4.0 Data Changes
+With the newest version of CIT, the process of importing data has been udpated. The data has been decoupled to a certain point where specific datasets have been organized into groups that can be run seperately. In the previous version, an Azure Container was running the import-data.sh script which ran the bootstrap command. Now that the data has been decoupled into groups, each group has its own command that can be run by executing:
+
```
Comment (bashbang) [inline DATA.md:3]: You speak to the "old way" using Azure container, but don't mention the "new way" of using GHA.  Did you want to include that?

File: cit-api/email.md [hunk 2]
```diff
@@ -16,4 +16,8 @@ To configure email notifications from the application, you must configure severa
...
\ No newline at end of file
+The third type of notification is a recurring notification sent to EDOs with opportunities that have not been modified in over 90 days. This is done through the [`send-reminders.sh`](./send-email-reminders.sh). When run, this script uses `manage.py` to run a python script that checks the last modified date for all opportunities in the database, and sends email notifications to the EDOs with submitted opportunities that have not been modified for 90 days. This script is run weekly in an Azure container that is started via an Azure Logic App to help ensure EDOs are keeping their opportunity information current.
+
```
Comment (bashbang) [inline cit-api/email.md:20]: Do you have plans to move this to GHA as well?

Comment (bashbang) [reply cit-api/email.md:20]: nm, I didn't read far enough.

Comment (bashbang) [reply cit-api/email.md:20]: Just need to add a Jira task to eventually obsolete the Azure containers once the GHA is fully implemented and tested.

**Review by bashbang — APPROVED** [review]
LGTM, just one comment about GHA in the Data Changes section you may wish to enhance.

File: cit-api/email.md (hunk 2 above)
Comment (jjstratton) [reply cit-api/email.md:20]: Okay sounds good!!

File: DATA.md (hunk 1 above)
Comment (jjstratton) [reply DATA.md:3]: I have it down a little further. Line 8 I think. Do you think thats too far?

**Pull request merged**